posthog/temporal/common/schedule.py

```diff
@@ -1,5 +1,22 @@
 from asgiref.sync import async_to_sync
-from temporalio.client import Client, Schedule, ScheduleUpdate, ScheduleUpdateInput
+from temporalio.client import Client, Schedule, ScheduleUpdate, ScheduleUpdateInput, ScheduleOverlapPolicy
+
+
+@async_to_sync
+async def trigger_schedule_buffer_one(temporal: Client, schedule_id: str):
+    """Trigger a Temporal Schedule using BUFFER_ONE overlap policy."""
+    handle = temporal.get_schedule_handle(schedule_id)
+    await handle.trigger(
+        overlap=ScheduleOverlapPolicy.BUFFER_ONE,
+    )
```

Comment:
I know you're following the convention in here already, but i'd just call `await a_trigger_schedule_buffer_one(...)` from this instead of duplicating the code between both functions